scripts/paxcheck.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+if ! type scanelf > /dev/null 2>&1; then
+    exit 0
+fi
```

Comment:
If you replicate this existence check in the Makefile.am then you could make this check fatal and issue a warning.  That would allow anyone running it manually to see the warning and it wouldn't cause a lint failure just because the tool wasn't installed.